iml-agent/src/cli.rs

```diff
@@ -233,6 +234,36 @@ pub enum StratagemClientCommand {
         #[structopt(flatten)]
         fidopts: FidInput,
     },
+
+    #[structopt(name = "filesync")]
+    /// Run FileSync action
+    FileSync {
+        #[structopt()]
```

Comment:
Is this needed?  Isn't an empty structopt just clutter?

Reply:
It is, but this can be cleaned up in a follow-on.